core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java

```diff
@@ -113,14 +111,16 @@ public ConsensusCommitConfig(DatabaseConfig databaseConfig) {
                 .toUpperCase(Locale.ROOT));
     if (isolation.equals(Isolation.SERIALIZABLE)) {
       validateCrossPartitionScanConfig(databaseConfig);
+
+      if (databaseConfig
+          .getProperties()
+          .containsKey("scalar.db.consensus_commit.serializable_strategy")) {
+        logger.warn(
+            "The property \"scalar.db.consensus_commit.serializable_strategy\" is deprecated and will "
+                + "be removed in 5.0.0. The EXTRA_READ strategy is always used for the SERIALIZABLE "
+                + "isolation level.");
+      }
```

Comment:
If the `scalar.db.consensus_commit.serializable_strategy` property is specified, a warning message will be shown.